lib/g5kchecks/utils/node.rb

```diff
@@ -39,7 +39,10 @@ def initialize(conf)
       @ohai_description = nil
       @api_description = nil
       @max_retries = 2
-      get_openstack_vendor_data
+      if @conf["noapi"] != true
```

Comment:
Could do `unless @conf["noapi"]`, it is equivalent in Ruby

Reply:
Implemented and tested.